plugins/mac-capture/mac-screen-capture.m

```diff
@@ -575,9 +593,12 @@ typedef void (^shareable_content_callback)(SCShareableContent *,
 	os_sem_wait(sc->shareable_content_available);
 	[sc->shareable_content release];
 	[SCShareableContent
-		getShareableContentExcludingDesktopWindows:excludingDesktopWindows
-				       onScreenWindowsOnly:!sc->show_hidden_windows
-					 completionHandler:new_content_received];
+		getShareableContentExcludingDesktopWindows:TRUE
```

Comment:
This will remove Finder, Dock, and Desktop from the list of shareable content - is that intended? Finder at the very least should be captureable.

Reply:
As discussed off-thread, this reinstates the code as it was before #6863, both on 12.5 and 13 Finder and Dock are capturable. The only thing missing is the wallpaper (and that's intentional).